docs/examples.md

```diff
@@ -228,3 +228,87 @@ cleanup() {
 setup
 #cleanup
 ```
+## Rancher for administering the cluster
+
+Install rancher in to the same cluser for administering the cluster.
+
+```
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: cattle-system
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rancher
+  namespace: cattle-system
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  # "namespace" omitted since ClusterRoles are not namespaced
+  name: rancher
+rules:
+- apiGroups: ["*"]
+  resources: ["*"]
+  verbs: ["*"]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+# This role binding allows "jane" to read pods in the "default" namespace.
+kind: ClusterRoleBinding
+metadata:
+  name: rancher
+subjects:
+- kind: ServiceAccount
+  name: rancher
+  namespace: cattle-system
+  apiGroup: ""
+roleRef:
+  kind: ClusterRole #this must be Role or ClusterRole
+  name: rancher
+  apiGroup: ""
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rancher
+  namespace: cattle-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: rancher
+  template:
+    metadata:
+      labels:
+        app: rancher
+    spec:
+      serviceAccountName: rancher
+      containers:
+      - name: name
+        image: rancher/rancher
+        resources:
+          requests:
+            cpu: 100m
+            memory: 200Mi
+        ports:
+        - containerPort: 443
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: rancher
+  namespace: cattle-system
+spec:
+  type: LoadBalancer
```

Comment:
If I'm not wrong, this will only work on a single-node k3d cluster, if you used `--no-deploy=traefik`, since otherwise, the node's port 443 will already be "taken" by traefik (or rather the svclb created for it).

Reply:
yeah that's correct. Do you think it'll be good to expose to some other port?
In that case, we'll have to change the initial `server-url` in rancher

Reply:
@iwilltry42 I updated the port to 8443, Now there won't be any blocker for `traefik`